package.json

```diff
@@ -11,16 +11,19 @@
     "declaration"
   ],
   "private": false,
-  "author": "philippsimon",
+  "author": "Digital UK",
+  "contributors": [
+    "philippsimon"
+  ],
```

Comment:
Would this not be the other way round? Digital UK are contributing to something originally created by philippsimon...

Reply:
> Would this not be the other way round? Digital UK are contributing to something originally created by philippsimon...

I got my steer [here](https://opensource.stackexchange.com/questions/8242/authors-and-contributors-of-forked-project).
I think this probably makes sense this way around. From hereon in, Digital UK is fundamentally in control of what's in and what's out.

Reply:
Am hanging on before merging though, in case there's any more debate to be had....

Reply:
The part I hadn't considered is that most people would look to the author to contact for issues and so the author should indeed be Digital-UK so I think this is good as is now.